rust/otap-dataflow/crates/otap/src/lib.rs

```diff
@@ -72,3 +72,61 @@ pub mod otlp_grpc;
 /// Factory for OTAP-based pipeline
 #[pipeline_factory(OTAP, OtapPdata)]
 pub static OTAP_PIPELINE_FACTORY: PipelineFactory<OtapPdata> = build_factory();
+
+#[cfg(test)]
+mod tests {
+    use otap_df_config::{
```

Comment:
nit: should this test be in the otap_df_controller crate?

It looks like we're only importing `otap_df_controller` crate in `otap_df_otap` for this test, but avoiding that will reserve for us the option to import `otap_df_otap` in the controller crate if we ever need to, without having a circular dependency

Reply:
I can move this to the controller crate, and let that crate's tests depend on otap-df-otap.